test/e2e/cluster_upgrade_runtimesdk.go

```diff
@@ -286,17 +271,19 @@ func extensionConfig(specName string, namespace *corev1.Namespace) *runtimev1.Ex
 			// when configuring external patches.
 			Name: specName,
 			Annotations: map[string]string{
-				runtimev1.InjectCAFromSecretAnnotation: fmt.Sprintf("%s/webhook-service-cert", namespace.Name),
+				runtimev1.InjectCAFromSecretAnnotation: "test-extension-system/test-extension-webhook-service-cert", // Note: this assumes the test extension get deployed in the default namespace defined in its own runtime-extensions-components.yaml
```

Comment:
nit: Can we move the note one line above in it's own line? This line is just getting a bit long without soft-wrap
(same below)